src/functions/forecast/list-project-sprints/handler.ts

```diff
@@ -40,7 +40,7 @@ const listProjectSprints = async (api: ForecastApiService, parameters: ListProje
     };
   });
 }
-
+// TODO: The related forecast endpoint is no longer used and so if this is still not needed when sprint view UI development is complete,
```

Comment:
add to the end "it should be removed".
I also think this should be moved to the first line of the file.